6_sixth_hw/4_fourth_task.py

```diff
@@ -0,0 +1,17 @@
+"""
+4. You have a file of unknown length. Write a function that will remove all numbers from each line of the file.
+"""
+import re
+
```

Comment:
it i task for abstract file.

So you should implement complex function of removing. 

with open("test_1.py", "r") as file:
        text_file = file.read()

and 

with open("test_1.py", "w") as file:
        file.write(remove_numbers(text_file))


should be incide function. And path to file is a parameter